tests/utils.py

```diff
@@ -13,14 +13,26 @@
 
 
 def xml_to_comparable_dict(xml):
-    def unorder(value):
+
+    def _sort_key(value):
+        """Recursively sort lists embedded within dicts."""
+        if hasattr(value, 'items'):
+            return six.text_type(sorted([(k, _sort_key(v)) for k, v in value.items()]))
```

Comment:
Is the reason for not using `iteritems` that this can be run in Python 2 as well as 3 ?